packages/itwin/reports-config-widget/src/widget/components/ExtractionToast.tsx

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { Text } from "@itwin/itwinui-react";
+import { ReportsConfigWidget } from "../../ReportsConfigWidget";
+
+interface ExtractionToastProps {
+  iModelName: string;
+  odataFeedUrl?: string;
+}
+
+export const SuccessfulExtractionToast = ({ iModelName, odataFeedUrl }: ExtractionToastProps) => {
+  return (
+    <div>
+      <Text>{ReportsConfigWidget.localization.getLocalizedString("ReportsConfigWidget:ExtractionSuccess")}{iModelName} </Text>
+      <a href="#" onClick={async () => { await navigator.clipboard.writeText(odataFeedUrl!); }}>{ReportsConfigWidget.localization.getLocalizedString("ReportsConfigWidget:CopyODataUrl")}</a>
```

Comment:
I feel like this line is too long. It's confusing to read at first.

Reply:
Move the `onClick` handler to a const and the contents of the `<a>` tag on a separate line. That should help with readability.

Reply:
`onClick` handler has been moved to a const. `<a>` tag contents separated